validate.go

```diff
@@ -80,17 +83,21 @@ var bundleValidateCommand = cli.Command{
 		}
 
 		hooksCheck := context.Bool("hooks")
-		bundleValidate(spec, rootfsPath, hooksCheck)
+		hostCheck := context.Bool("host-specific")
+		bundleValidate(spec, rootfsPath, hooksCheck, hostCheck)
 		logrus.Infof("Bundle validation succeeded.")
 		return nil
 	},
 }
 
-func bundleValidate(spec rspec.Spec, rootfs string, hooksCheck bool) {
+func bundleValidate(spec rspec.Spec, rootfs string, hooksCheck bool, hostCheck bool) {
 	checkMandatoryField(spec)
 	checkSemVer(spec.Version)
 	checkPlatform(spec.Platform)
```

Comment:
On Mon, Jun 27, 2016 at 07:27:17AM -0700, Ma Shimiao wrote:
> >  	checkPlatform(spec.Platform)
> 
> I'm not sure what spec means. I think that's runtime's
> business. checkPlatform seems can't judge you mentioned above.  Or
> could you give me more detailed information about how to modify
> checkPlatform?

Let's just punt this to a follow-up PR.